### PR TITLE
Update CHANGELOG for beta release

### PR DESCRIPTION
**Pull request by jonahx**
#### What does this PR do (include background context, if relevant)?

This is the PR for the beta release.  It includes work from the past 2 months, such as refactoring, but is mainly geared at providing doc updates, XA assurance, and other updates that make Secretless ready for beta testing by our enterprise partners.

#### What ticket does this PR close?

It's related to #643

#### Where should the reviewer start?

N/A

#### What is the status of the manual tests?

Have you run the following manual tests to verify existing functionality continues to function as expected?
- [x] Manually tested [K8s CRDs](https://github.com/cyberark/secretless-broker/tree/master/test/manual/k8s_crds)
- [x] Manually tested [Keychain provider](https://github.com/cyberark/secretless-broker/tree/master/test/manual/keychain_provider)
- [x] Manually run the [K8s demo](https://github.com/cyberark/secretless-broker/tree/master/demos/k8s-demo)
- [x] Manually run [Kubernetes-Conjur demo](https://github.com/conjurdemos/kubernetes-conjur-demo) with a local Secretless Broker image build of your branch
- [ ] Manually run the [full demo](https://github.com/cyberark/secretless-broker/tree/master/demos/full-demo) (optional)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,82 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.7.0] 2019-03-26
+
+### TODO: Don't commit this without proofreading!!!
```
Comment (izgeri) [inline CHANGELOG.md]: D'oh

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,82 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+### Added
+
+### Changed
+
+- Merge pull request #669 from cyberark/doodlesbykumbi-patch-1
```
Comment (izgeri) [inline CHANGELOG.md]: This needs to be cleaned up. Aggregating and summarizing things based on the commit history is fine, and even preferable. There should be no `Merge pull request` lines.

File: CHANGELOG.md [hunk 3]
```diff
@@ -203,3 +279,5 @@ The first tagged version.
 [0.6.2]: https://github.com/cyberark/secretless-broker/compare/v0.6.1...v0.6.2
 [0.6.3]: https://github.com/cyberark/secretless-broker/compare/v0.6.2...v0.6.3
 [0.6.4]: https://github.com/cyberark/secretless-broker/compare/v0.6.3...v0.6.4
+[0.7.0]: https://github.com/cyberark/secretless-broker/compare/v0.6.4...v0.7.0 
+
```
Comment (izgeri) [inline CHANGELOG.md]: This is a bug in the script, and this line needs to be removed

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,8 +6,48 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ### Added
-- Added ability to verify plugin checksums
+
+- Add ability to verify plugin checksums
+- Add kubernetes secrets provider to README.md
+- Add a markdown file for k8s secrets provider
+- Add link to /tutorials in the top nav
+- Note styling in Kubernetes tutorial
+- Add link to /tutorials in the top nav
```
Comment (izgeri) [inline CHANGELOG.md]: Duplicate

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,8 +6,48 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ## [Unreleased]
 
+## [0.7.0] 2019-03-26
+
 ### Added
-- Added ability to verify plugin checksums
+
+- Add ability to verify plugin checksums
+- Add kubernetes secrets provider to README.md
+- Add a markdown file for k8s secrets provider
```
Comment (izgeri) [inline CHANGELOG.md]: this one is canceled out by the one below, please remove

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,8 +6,48 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Add daily build trigger
+- Add redirect link capabilities
+- Add version to README.md
+- Add a README for the shared library
+- C shared library exposing secret providers (POC)
+- Add custom 404 page
+
+### Changed
+
+- Remove kubernetes secrets page
```
Comment (izgeri) [inline CHANGELOG.md]: this one is canceled out by the one above, please remove

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,8 +6,48 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Update vault command to add secrets in test
```
Comment (izgeri) [inline CHANGELOG.md]: this is a fix - fixed the vault test that was broken due to vault CLI updates

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,8 +6,48 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Re-enabled the ssh-handler tests
+- Unlocked the root account in `ssh_handler` test ssh host container
```
Comment (izgeri) [inline CHANGELOG.md]: this is a fix, and goes with the one above it

**Comment by jonahx** [comment]
> A few final nits

always encouraged!

**Pull request merged**